system/helpers/artefactkeygenerator/golang-project-repository-commit.py

```diff
@@ -0,0 +1,27 @@
+from infra.system.core.meta.metaartefactkeygenerator import MetaArtefactKeyGenerator
+from infra.system.artefacts.artefacts import ARTEFACT_GOLANG_PROJECT_REPOSITORY_COMMIT
+import logging
+
+class GolangProjectRepositoryCommitKeyGenerator(MetaArtefactKeyGenerator):
+	"""
+	{
+	"artefact": golang-project-repository-commit,
+	"repository": "github.com/coreos/etcd",
+	"commit": "729b530c489a73532843e664ae9c6db5c686d314"
+	"adate": timestamp,
+	"cdate": timestamp,
+	"author": "Gyu-Ho Lee <gyuhox@gmail.com>",
+	"message": "commit message ..."
+	}
+	"""
+	def generate(self, data, delimiter=":"):
+		# return a list of fields
+		keys = []
+		for key in ["artefact","repository", "commit","adate", "cdate", "author", "message"]:
```

Comment:
Only "artefact", "repository", "commit" are needed to uniquely determine entry in database.
Please, update [1] and [1] with new artefacts and their keys as well.

[1] https://github.com/gofed/infra/blob/master/generators/artefacts.json
[2] https://github.com/gofed/infra/blob/master/system/artefacts/artefacts.py